packages/ketcher-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ketcher-core",
-  "version": "2.6.4",
+  "version": "2.7.0-rc.4",
```

Comment:
Are you sure, we need this? 
In master we still have 2.6.4, in release there is a 2.7.0-rc.8